autovalet_arucotags/aruco_gazebo/.gitignore

```diff
@@ -0,0 +1,2 @@
+#Temp
+*~
```

Comment:
NIT: remove this file since it is not necessary once we deleted the `.git` folder

Reply:
Done